File: arrhash/twoSum.js
let nums = [3, 4, 5, 6];
let target = 7;

function twoSum(nums, target) {
  hm = {};
  for (let i = 0; i < nums.length; i++) {
    complement = target - nums[i];
    if (complement in hm) {
      return [hm[complement], i];
    } else {
      hm[nums[i]] = i;
    }
  }
}

const result = twoSum(nums, target);

document.getElementById("result").textContent = "Resultado: " + result;

File: arrhash/twoSum.py
nums = [4, 5, 6]
target = 10

def twoSum(nums, target):
  hm = {}
  for i, n in enumerate(nums):
    complement = target - n
    if complement in hm:
      return [hm[complement], i]
    else:
      hm[n] = i

result = twoSum(nums, target)
print(result)
